app/assets/stylesheets/application.css.scss

```diff
@@ -18,3 +18,163 @@
 @import "bootstrap";
 @import "font-awesome-sprockets";
 @import "font-awesome";
+
+body {
+  font-family: "century gothic", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #333333;
+  background-color: #ebe7df
+}
+
+.row h1 {
+  font-weight:bold;
+  font-size:20px;
+  background:#418a95;
+  padding:30px 10px;
+  color:#fff;
+  margin-bottom:0;
+}
+
+.table {
+  width: 100%;
+  max-width: 100%;
+  margin-bottom: 20px;
+  box-shadow:0px 10px 10px rgba(0, 0, 0, 0.2);
+}
+.table tr {
+  border-top:1px solid #ebe7df;
+}
+.table thead tr {
+  border-bottom:1px solid #ebe7df;
+  border-top:1px solid #63acb7;
+  background:#63acb7;
+  color:#fff;
+}
+.table thead tr:first-child {
+  border-top:none
```

Comment:
Colon after property should be followed by one space<br>Declaration should be terminated by a semicolon